packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneButton/theme.js

```diff
@@ -24,7 +22,7 @@ const theme = {
 
       &:focus:not(:disabled),
       &:hover:not(:disabled) {
-        background: ${DARK_GREEN};
+        background: ${darken(0.15, props.theme.global.colors['status-ok'])};
         box-shadow: none;
```

Comment:
`box-shadow: none;` should be removed

Reply:
I remember why I did this - the default Grommet button theme uses `box-shadow` to create an extra border on focus / hover